src/js/experiments/gettingStartedGuide.js

```diff
@@ -29,19 +25,59 @@ function showHomePageBadges() {
   }
 }
 
-// NOTE: experimentContainer can be updated once we have the other components of this experiment created and in place
 window.OptimizelyClient.getVariationName({
   experimentKey: 'dd_getting_started_docs_test',
   groupExperimentName: 'q1_fy23_docs_disco_experiment_group_test',
   experimentContainer: 'body',
   guestExperiment: false,
 }).then((variation) => {
   if (variation === 'treatment') {
+    // Used to expand the container width for the content
+    if (window.location.pathname === '/docs/2.0/getting-started/') {
+      const articleContainer = $('.quickstart-guide');
+      articleContainer.addClass('getting-started-full-width');
+    }
+
+    // Init new badge in sidebar and add tracking event
     addNewBadgeToSidebar();
+
+    // Init new badge on landing page and add tracking events
     showHomePageBadges();
-    // console.log('IN TREATMENT GROUP');
+    const badges = Array.from($('.wrapper-link'));
+    badges.forEach((badge) => {
+      badge.addEventListener('click', () => {
+        window.AnalyticsClient.trackAction('clicked-landing-page-badge', {
+          badgeText: badge.innerText,
+          badgeHref: badge.href,
+        });
+      });
+    });
```

Comment:
I recommend that we move this code and the code for tracking Links into a single function: `setupTracking`.

It's somewhat a preference thing - but imo,  it's easier to scan the bottom of the file in the treatment block and just see a list of named functions - this enables someone coming to read the code in the future to not have to scan each block of code to know what it does before realizing that it probably isn't what they need to change - vs - reading 4 lines of function names and (hopefully, if they are well named) jump to that function and make the changes they need to make.

Reply:
great call out! I refactored the code for tracking and added in `setupTracking` at the bottom. I think once we merge this branch into [DD-450](https://github.com/circleci/circleci-docs/pull/6544) we should be good to merge that into master when we want to start running the experiment.